Makefile

```diff
@@ -156,7 +156,8 @@ rust-test:
 	+$(CARGO) test $(CARGO_TEST_FLAGS) --no-run
 
 test/test: xlicense.h $(LIB_OUTPUT) $(NEWSBOATLIB_OUTPUT) $(NEWSBOAT_OBJS) $(PODBOAT_OBJS) $(FILTERLIB_OUTPUT) $(RSSPPLIB_OUTPUT) $(TEST_OBJS)
-	$(CXX) $(CXXFLAGS) -o test/test $(TEST_OBJS) $(SRC_OBJS) $(NEWSBOAT_LIBS) $(LDFLAGS)
+	$(CXX) $(CXXFLAGS) -Wno-double-promotion -MD -MP -o 3rd-party/catch.o -c 3rd-party/catch.cpp
```

Comment:
Nit: does it take long to compile? I'd rather not re-compile the entire framework every time I change a single test.

Reply:
Fixed!